src/zoid/content.js

```diff
@@ -0,0 +1,162 @@
+/* @flow */
```

Comment:
Refactored out of checkout to keep size down and be reused by both PayPal and Venmo.